docs/source/using/profiles.adoc

```diff
@@ -43,16 +59,17 @@ $ minishift profile list
 [[switching-profiles]]
 == Switching Profiles
 
-With the help of xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`] you can switch between profiles or set a profile as active.
-When a profile is active, it is not required to pass `--profile` flag to run minishift commands against that profile.
+You can switch between profiles with the help of xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`] command.
+At any time only one profile can be the active profile.
```

Comment:
This is technically correct, but awkward to read. Suggested rewrite: "Only one profile can be active at any time."

If it's more technically accurate to specifically reference "the active profile" versus "one profile can be active", we can simply move "at any time" to the end of the sentence rather than the beginning.